libs/runtime/ebpf_bitmap.c

```diff
@@ -100,6 +103,26 @@ ebpf_bitmap_start_forward_search(_In_ const ebpf_bitmap_t* bitmap, _Out_ ebpf_bi
     internal_cursor->current_block_copy = bitmap->data[BIT_TO_BLOCK(internal_cursor->next_bit_offset)];
 }
 
+void
+ebpf_bitmap_start_forward_search_at(
```

Comment:
This looks like new logic.  [The project contributing guidelines](https://github.com/microsoft/ebpf-for-windows/blob/main/CONTRIBUTING.md#contributing-code) state:
> For all but the absolute simplest changes, first [submit a GitHub Issue](https://github.com/microsoft/ebpf-for-windows/issues/new)

What issue is this addressing?

Reply:
`reverse_search_at` was added to fix some of the Heap Buffer Overflows in _ebpf_murmur3_32 mentioned in #4010 

The forward search function was just added at the same time, but is not currently needed (could be removed).

The problem was that reverse search could return a bit past the key length, so to initialize search at a point other than the start/end we need to clear the bits before/after the start index in the start block.

Reply:
I have removed the dead code `_forward_search_at`.